charts/internal/seed-controlplane/charts/csi-driver-controller/values.yaml

```diff
@@ -73,4 +91,4 @@ csiSnapshotValidationWebhook:
       cpu: 10m
       memory: 32Mi
     limits:
-      memory: 200Mi 
+      memory: 200Mi
```

Comment:
Is it intentional that this is missing `maxAllowed` ?

Reply:
The [csi-driver-node](https://github.com/gardener/gardener-extension-provider-gcp/blob/8226169d598536b4384b13fcd7de5710caafb836/charts/internal/shoot-system-components/charts/csi-driver-node/templates/vpa.yaml#L5) on shoot is also marked as critical component. Thanks for pointing it out, I'll update the PR.